src/pages/index.tsx

```diff
@@ -1,33 +1,174 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import Nav from 'components/Nav';
-// import InkLogo from '../assets/InkLogo';
-
-// import { Meta } from '../layout/Meta';
-// import { Main } from '../templates/Main';
-
-const Index = () => (
-  // <Main
-  //   meta={(
-  //     <Meta
-  //       title="ink Society"
-  //       description="ink Society is a bookclub management site"
-  //     />
-  //   )}
-  // >
-  //   <h1 className="font-bold text-2xl">
-  //     Boilerplate code for your Nextjs project with Tailwind CSS
-  //   </h1>
-  // </Main>
-  <div className="h-screen bg-black">
-    <Nav />
-    {/* <InkLogo/> */}
-    <div className="absolute bottom-0 left-0 m-8 md:m-16 h-1/4">
-      <h1 className="text-white text-3xl md:text-6xl extrabold">
-        The last bookclub manager you&apos;ll need.
-      </h1>
+
+// import InkLogo from '../assets/InkLogo.svg';
+
+const Index = () => {
+  useEffect(() => {
+    function scrollFunction() {
+      if (document.body.scrollTop > 20 || document.documentElement.scrollTop > 20) {
+        document.getElementById('navbar')!.style.top = '0';
+      } else {
+        document.getElementById('navbar')!.style.top = '-300px';
+      }
+    }
+    if (window !== 'undefined') {
+      window.onscroll = () => {
+        scrollFunction();
+      };
+    }
+  });
+
+  return (
+    <div>
+      <Nav />
+      <div className="h-screen">
+        <div>{/* <InkLogo className="max-h-1/2" /> */}</div>
+        <div>
+          <div className="absolute text-black bottom-0 left-0 m-8 md:m-16 h-1/6 typewriter">
+            <h1 className="text-3xl md:text-6xl extrabold ">The last bookclub</h1>
+            <h1 className="text-3xl md:text-6xl extrabold ">manager you&apos;ll need.</h1>
+          </div>
+        </div>
+      </div>
+      <div className="py-12 bg-white">
```

Comment:
Do we need to check in all the boilerplate code or was it just for testing?

Reply:
I will be checking all tailwind and TS boilerplate code in my next branch. However I did go ahead and remove the white bg as the body is already white.